apps/docs/package.json

```diff
@@ -39,7 +39,7 @@
     "react-helmet-async": "^1.0.7",
     "react-router": "^5.2.0",
     "react-router-dom": "^5.2.0",
-    "serverless": "^1.75.1",
+    "serverless": "^2.62.0",
```

Comment:
Any chance of testing a ~possible~ _probable_ breaking change? Or can this only be done by Cypress tests running against the PR prototype URL?.. which, is that even setup yet!?

Reply:
It deploys to Netlify so it seems to be working. ;-)